awx/settings/defaults.py

```diff
@@ -1089,6 +1095,9 @@
 # Older records will be cleaned up
 INDIRECT_HOST_AUDIT_RECORD_MAX_AGE_DAYS = 7
 
+# setting for Policy as Code feature
+FEATURE_POLICY_AS_CODE_ENABLED = False
```

Comment:
depends on if policy as code is supposed to be in awx

Reply:
Another good reference on this https://github.com/ansible/awx/pull/15986, was intended to be turned on then, but got abandoned.

Reply:
Turns on the forward-port is already done, we're good.